app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # :nodoc:
+    class CommentsController < ApplicationController
+      def index
+        Comment.all
```

Comment:
Why do you use Comment.all on the first line of this action?
You can use only
def index
  render json: Comment.all, message: 'Comment list', status: :ok
end